packages/react-aria-components/src/index.ts

```diff
@@ -16,6 +16,8 @@ import 'client-only';
 
 export {CheckboxContext, ColorAreaContext, ColorFieldContext, ColorSliderContext, ColorWheelContext, HeadingContext} from './RSPContexts';
 
+// TODO: export the respective contexts here
+export {Autocomplete} from './Autocomplete';
```

Comment:
Waiting for the api to settle. Right now there are three contexts in Autocomplete:  AutocompleteContext (standard one for users to pass stuff from an external component to the Autocomplete component), AutocompleteStateContext (also standard, for accessing the state from within the component), InternalAutocompleteContext (for accessing the stuff Autocomplete wants to pass to its wrapped collection components)